REFERENCE.md

```diff
@@ -40,6 +40,8 @@ The following parameters are available in the `profile_rsyslog` class:
 * [`feature_packages`](#feature_packages)
 * [`override_default_config`](#override_default_config)
 * [`purge_config_files`](#purge_config_files)
+* [`systemd_journald_settings,`](#systemd_journald_settings,)
```

Comment:
Why is this same parameter referenced twice?

Reply:
Evidently because in the '@param' line I included a comma after the variable name. When I removed that it removed the duplicate (that also had a comma at the end).